velox/exec/HashProbe.cpp

```diff
@@ -516,7 +509,7 @@ RowVectorPtr HashProbe::getOutput() {
     if (isRightJoin(joinType_) || isFullJoin(joinType_) ||
         isRightSemiJoin(joinType_)) {
       // Mark build-side rows that have a match on the join condition.
-      table_->rows()->setProbedFlag(outputRows_.data(), numOut);
+      table_->rows()->setProbedFlag(outputTableRows_.data(), numOut);
```

Comment:
Hi @xiaoxmeng , seems multiple drivers/threads might concurrently call `setProbedFlag` and cause data race here? cc @Yuhta @mbasmanova

Reply:
True, but since all these threads are setting the flag to the same value (true), then it is ok, no?

Reply:
Yes, I think it's ok for such case.